packages/manager/tools/webpack-dev-server/src/config.js

```diff
@@ -30,7 +30,7 @@ export = (env) => {
     proxy.unshift(serverProxy.aapi);
   }
   if (env.dev) {
-    proxy.unshift(...env.dev.map((config) => serverProxy.dev(config)));
+    proxy.unshift(...env.dev.map((c) => serverProxy.dev(c)));
```

Comment:
Issue with leaving `config` as the name of parameter ?

Reply:
Yep, it gives the error "config is already declared in the upper scope. 

Although, now that i re-read the code, i think i can rename it "config" and remove the upper-scope object exported to export it directly ! Thanks !